test/example/qt.erl

```diff
@@ -457,6 +484,28 @@ check_processes() ->
   end.
 
 
+perf(S, N, Packet) ->
+  Sz = size(Packet),
+  {ok, 12} = quicer:send(S, <<"perf", Sz:32, N:32>>),
+  {ok, <<"ok">>} = quicer:recv(S, 2),
+  perf2(S, N, Sz, Packet).
+
+perf2(_S, 0, _, _) ->
+  ok;
+perf2(S, N, Sz, Packet) ->
+  {ok, Sz} = quicer:send(S, Packet),
+  {ok, Bin} = quicer:recv(S, Sz),
+  Sz = size(Bin),
+  perf2(S, N-1, Sz, Packet).
+
+
+
+pack(N) ->
+  list_to_binary(
```

Comment:
Wonder if you map all values to `18`? do you meet issue with incomplete/corrupted data?
I am asking to ensure you don't get the above issues.


nit, if you want an increasing integers binary stream,  here is a simplified version.
```
<< << A >> || A <- lists:seq(1, N) >>.
```